### PR TITLE
changes data to reflect american banks and names for HCL workshop

### PR DESCRIPTION
**Pull request by sanjeevghimire**
* Data changes

### CODE REVIEW

File: data/conversation/workspaces/banking.json [hunk 1]
```diff
@@ -1 +1 @@
...11196","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"@features:status","description":null,"dialog_node":"node_25_1486267658814","previous_sibling":"node_26_1486269385463"},{"go_to":null,"title":null,"output":{"text":{"values":["Please contact the Merchant Customer Care for queries related to payment and product."],"selection_policy":"sequential"}},"parent":"node_5_1486279608967","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"@features:product || @option:payment || @account_type:merchant","description":null,"dialog_node":"node_10_1486280199008","previous_sibling":"node_7_1486279676925"},{"go_to":null,"title":null,"output":{"text":{"values":["Please enter a valid PAN number."],"selection_policy":"sequential"}},"parent":"node_1_1492426752086","context":null,"created":"2017-06-12T18:08:50.365Z","updated":"2017-06-12T18:08:50.365Z","metadata":null,"conditions":"true","description":null,"dialog_node":"node_11_1492429161945","previous_sibling":"node_3_1492427252795"},{"go_to":null,"title":null,"output":{"text":{"values":["Please enter the amount.
```
Comment (stevemart) [inline data/conversation/workspaces/banking.json]: the beginning of this file has some garbage we can probably trim out

```
{"intent":"nonsense","examples":[{"text":"ajshajhds"},{"text":"is the sky blue"},{"text":"hashahah"},{"text":"hahahaha"},{"text":"hahahah"},{"text":"come here"},{"text":"ccc"},{"text":"buffalo"},{"text":"are you mad"},{"text":"are you a fool"},{"text":"this is red"},{"text":"sdjhsjd"},{"text":"savings"},{"text":"it is bad"}],"description":null}
```

File: banking_services.js [hunk 2]
```diff
@@ -18,15 +18,15 @@
...
+    fname: 'John',
+    lname: 'Doe',
     address: {
-      line1: 'B 101 Amar CHS Ltd',
-      line2: 'Bandra West',
-      city: 'Mumbai',
-      state: 'MH',
-      zip: 400001,
-      country: 'India'
+      line1: '3028 Austin st.',
```
Comment (dolph) [inline banking_services.js]: Captalize the abbreviation: `St.`

File: banking_services.js [hunk 3]
```diff
@@ -18,15 +18,15 @@
...
+      line2: 'Apt #top floor',
```
Comment (dolph) [inline banking_services.js]: Pick a number?

File: banking_services.js [hunk 4]
```diff
@@ -268,32 +268,32 @@ const bankingServices = {
...
+      address: '13501 Burned Rd, Austin, TX, 78759',
+      phone: '512-347-1200',
       hours: '10AM–4PM'
     },
     {
-      location: 'andheri',
-      address: 'Swami Vivekanand Road, Andheri West, Mumbai, Maharashtra 400058',
-      phone: '022 0000 0000',
+      location: 'san fransisco',
+      address: '413 Market St, San Fransisco, CA, 94105',
```
Comment (dolph) [inline banking_services.js]: Francisco*

File: banking_services.js [hunk 5]
```diff
@@ -268,32 +268,32 @@ const bankingServices = {
...
+      location: 'Austin',
+      address: '13501 Burned Rd, Austin, TX, 78759',
+      phone: '512-347-1200',
       hours: '10AM–4PM'
     },
     {
-      location: 'andheri',
-      address: 'Swami Vivekanand Road, Andheri West, Mumbai, Maharashtra 400058',
-      phone: '022 0000 0000',
+      location: 'san fransisco',
```
Comment (dolph) [inline banking_services.js]: Francisco*

File: data/conversation/workspaces/banking.json (hunk 1 above)
Comment (dolph) [reply data/conversation/workspaces/banking.json]: ++ the conditional referencing `#nonsense` would need to be revised as well.

File: banking_services.js [hunk 6]
```diff
@@ -268,32 +268,32 @@ const bankingServices = {
...
+      address: '413 Market St, San Fransisco, CA, 94105',
+      phone: '415-213-4567',
       hours: '10AM–5PM'
     },
     {
-      location: 'delhi',
-      address: 'Rohini, Delhi, 110042',
-      phone: '011 0000 0000',
+      location: 'seattle',
+      address: '1200 5th Ave, Seattle, WA',
```
Comment (dolph) [inline banking_services.js]: Address for an IBM Building

File: banking_services.js [hunk 7]
```diff
@@ -268,32 +268,32 @@ const bankingServices = {
       hours: '10AM–4PM'
     },
     {
-      location: 'cochin',
-      address: 'Cochin Shipyard, Mahatma Gandhi Rd, Ernakulam, Kerala 682015',
-      phone: '0484 0000 0000',
+      location: 'Austin',
+      address: '13501 Burned Rd, Austin, TX, 78759',
+      phone: '512-347-1200',
```
Comment (dolph) [inline banking_services.js]: This is the phone number for a real (in)famous personal injury attorney.

File: banking_services.js [hunk 8]
```diff
@@ -268,32 +268,32 @@ const bankingServices = {
...
+      address: '1200 5th Ave, Seattle, WA',
+      phone: '425-666-0000',
       hours: '10AM–5PM'
     },
     {
-      location: 'bangalore',
-      address: 'Bangalore Airport Exit Road, Bengaluru, Karnataka 560017',
+      location: 'florida',
+      address: '112 Florida st, Miami, Fl, 12345',
       phone: '080 0000 0000',
```
Comment (dolph) [inline banking_services.js]: Not a North American phone format.

File: banking_services.js [hunk 9]
```diff
@@ -268,32 +268,32 @@ const bankingServices = {
...
       hours: '10AM–5PM'
     },
     {
-      location: 'mumbai',
-      address: 'Nariman point, Mumbai, Maharashtra 400023',
+      location: 'boston',
+      address: '1234 Boston St, Boston, MA 65320',
       phone: '022 1111 1111',
```
Comment (dolph) [inline banking_services.js]: Not a North American phone format.

**Comment by markstur** [comment]
This has sat for quite a while and still conflicts with the India use case.  I'll close the PR, but keep the branch.

**Pull request closed without merging**